src/Line/Provider.php

```diff
@@ -12,16 +12,12 @@ class Provider extends AbstractProvider
     public const IDENTIFIER = 'LINE';
 
     /**
-     * The separating character for the requested scopes.
-     *
-     * @var string
+     * {@inheritdoc}
      */
     protected $scopeSeparator = ' ';
 
     /**
-     * The scopes being requested.
-     *
-     * @var array
+     * {@inheritdoc}
```

Comment:
Why is this needed? This is just for code doc generation right, which we dont do?

Reply:
> This is just for code doc generation right, which we dont do?
It is

Another approach would be be to remove all the docblocks

Reply:
@atymic I can remove them all, just let me know

Reply:
I've removed them; see 3de8aca127600ef506df292e3ca66d06a62cb202